__tests__/app.test.js

```diff
@@ -41,6 +41,50 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles", () => {
+  describe("GET /api/articles/:article_id", () => {
+    test("GET 200: Responds with the required article object", () => {
+      return request(app)
+        .get("/api/articles/3")
+        .expect(200)
+        .then((response) => {
```

Comment:
I would destructure the response to {body} so we dont have to write so much below